File: app/src/main/java/fansirsqi/xposed/sesame/entity/OtherEntity.java
package fansirsqi.xposed.sesame.entity;

import java.util.ArrayList;
import java.util.List;

public class OtherEntity extends ForestEntity {
    public OtherEntity(String i, String n) {
        super(i, n);
    }

    public static List<OtherEntity> listEcoLifeOptions() {
        List<OtherEntity> list = new ArrayList<>();
        list.add(new OtherEntity("tick", "绿色行动"));
        list.add(new OtherEntity("plate", "光盘行动"));
        return list;
    }

    public static List<OtherEntity> listHealthcareOptions() {
        List<OtherEntity> list = new ArrayList<>();
        list.add(new OtherEntity("FEEDS", "绿色医疗💉"));
        list.add(new OtherEntity("BILL", "电子小票🎫"));
        return list;
    }


}

File: app/src/main/java/fansirsqi/xposed/sesame/entity/OtherEntity.kt
package fansirsqi.xposed.sesame.entity

class OtherEntity(id: String, name: String) : MapperEntity() {
    init {
        this.id = id
        this.name = name
    }
}

object OtherEntityProvider {
    @JvmStatic
    fun listEcoLifeOptions(): List<OtherEntity> = listOf(
        OtherEntity("tick", "绿色行动🍃"),
        OtherEntity("plate", "光盘行动💽")
    )

    @JvmStatic
    fun listHealthcareOptions(): List<OtherEntity> = listOf(
        OtherEntity("FEEDS", "绿色医疗💉"),
        OtherEntity("BILL", "电子小票🎫")
    )

    @JvmStatic
    fun farmFamilyOption():List<OtherEntity> = listOf(
        OtherEntity("familySign", "每日签到📅"),
        OtherEntity("assignRights", "使用顶梁柱特权👷‍♂️"),
        OtherEntity("familyClaimReward", "领取奖励🏆️"),
        OtherEntity("feedFamilyAnimal", "帮喂小鸡🐔"),
        OtherEntity("eatTogetherConfig", "请吃美食🍲"),
        OtherEntity("deliverMsgSend", "道早安🌞"),
//        OtherEntity("inviteFriendVisitFamily", "好友分享🙆‍♂️"),
//        OtherEntity("familyDrawInfo", "开扭蛋🩲"),
//        OtherEntity("batchInviteP2P", "串门送扭蛋🥚")
    )
}